File: BACKEND/Siscon_API_Backend/Siscon_API_Backend/Models/Usuario.cs
namespace Siscon_API_Backend.Models
{
    public class Usuario
    {
        public long Id { get; set; }
        public string LoginNome { get; set; }
        public string Senha { get; set; }
        public string NomeUsuario { get; set; }
        public DateTime? DataCriacao { get; set; }
    }
}

File: FRONTEND/SisconFrontEnd/SisconFrontEnd/src/app/models/Usuario.ts
export class Usuario {

  usuario: string;

  senha: string;

  nome_usuario: string | "";

  dataCriacao: string | "";

}

export class UsuarioPayload {
  data: Usuario[];
}

export class UsuarioPayloadUniqueResult {
  data: Usuario;
}

export class ValidaSenhaUsuarioPayload {
  data: ValidaSenhaUsuario ;
}

export class ValidaSenhaUsuario {
  usuario: string;
  loginValidado!: boolean;
}
